capstone-frontend/src/components/SearchUserModal.jsx

```diff
@@ -15,14 +15,12 @@ export class SearchUserModal extends Component {
       typingTimeout: 0,
       searchTerm: '',
       searchResults: [],
-      addedUsersTracker: [],
       addedUsers: [],
```

Comment:
Nice job simplifying the logic here to just using the single `addedUsers` variable to keep track of added club members and added friends!